src/Goteo/Model/Node/NodeSections.php

```diff
@@ -58,31 +60,26 @@ public static function getLangFields() {
 
 
   /**
-   * Get data about Node Sections
-   *
-   * @param   int    $id         check id.
-   * @return  NodeSections object
+   * @return  NodeSections
```

Comment:
This @return comments isn't necessary, since there's already a return parameter in this method